.github/workflows/build.yml

```diff
@@ -31,11 +31,11 @@ jobs:
             ~/.gradle/wrapper/'
           key: ${{ runner.os }}-mvn_sonar_gradle_cache
 
-      - name: Set up JDK 8
+      - name: Set up JDK 16
```

Comment:
Apologies I'm not fluent in Kolten; Is the name on line 20 still correct, or is the project getting built with java 16 now?

Reply:
Yes the project is now built using java 16.  In the discord I was told that this is something that was planned anyways. If desired I can build the engine itself with an older version. Though for `utiliti` it is easier if `jpackage` is available.

It is configured here:
https://github.com/weisJ/litiengine/blob/6c2023cc5d9883bd41ab175cfd48e88f069b12c4/build.gradle.kts#L60

Notably if no matching compiler and jdk is available gradle will download an appropriate version itself.

Reply:
Line #20 should also be updated to say `Java 16 build` instead of `Java 8 build` then.

Reply:
Fixed